modules/drivers/bigquery/src/metabase/driver/bigquery.clj

```diff
@@ -506,14 +542,17 @@
     (time/time-zone-for-id (.getID jvm-tz))
     time/utc))
 
+(defn- paginate? [{{context :context} :info}]
+  (contains? #{:csv-download :json-download :xlsx-download} context))
+
 (defmethod driver/execute-query :bigquery [driver {{sql :query, params :params, :keys [table-name mbql?]} :native
-                                                   :as                                                    outer-query}]
+                                                   :as                                                    query-request}]
   (let [database (qp.store/database)]
     (binding [*bigquery-timezone* (effective-query-timezone database)]
-      (let [sql (str "-- " (qputil/query->remark outer-query) "\n" (if (seq params)
-                                                                     (unprepare/unprepare driver (cons sql params))
-                                                                     sql))]
-        (process-native* database sql)))))
```

Comment:
So... we can't just remove this remark like that. A lot of people rely on it to see who in their company is running too many queries or things like that. I didn't put it in there just for fun 😉 

It seems like half of Metabase + BigQuery users want this out because it breaks caching, while the other half want it in for auditing purposes. I've said this in the past but I'll merge a PR that makes this option configurable, but not one that removes it entirely